tests/integration.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+mod tests {
+    use anyhow::Result;
+    use image_rs::image::ImageClient;
+    use std::path::Path;
+    use std::process::{Child, Command};
+
+    fn start_attestation_agent() -> Result<Child> {
+        let script_dir = format!("{}/{}", std::env!("CARGO_MANIFEST_DIR"), "scripts");
```

Comment:
Not a rust expert but I would use https://doc.rust-lang.org/std/path/struct.Path.html to manipulate paths. Not a big deal, it is up to you take this suggestion :)